lib/pacemaker/pcmk_graph_producer.c

```diff
@@ -420,7 +420,11 @@ create_graph_action(xmlNode *parent, pe_action_t *action, bool skip_details,
             needs_maintenance_info = true;
         }
         action_xml = create_xml_node(parent, XML_GRAPH_TAG_PSEUDO_EVENT);
-        needs_node_info = false;
+
+        if (action->rsc && !action->rsc->is_remote_node &&
+            pcmk__str_eq(action->task, RSC_STOP, pcmk__str_none)) {
+            needs_node_info = false;
+        }
```

Comment:
This isn't quite right ... we want it to be false for all pseudo-actions except the ones we need it on

Reply:
Fixed, I think.